docs/sources/variables/templates-and-variables.md

```diff
@@ -54,196 +68,48 @@ documentation topic for details on value escaping during interpolation.
-
-### Query options
-
-This variable type is the most powerful and complex as it can dynamically fetch its options using a data source query.
-
-Option | Description
-------- | --------
-*Data source* | The data source target for the query.
-*Refresh* | Controls when to update the variable option list (values in the dropdown). **On Dashboard Load** will slow down dashboard load as the variable query needs to be completed before dashboard can be initialized. Set this only to **On Time Range Change** if your variable options query contains a time range filter or is dependent on dashboard time range.
-*Query* | The data source specific query expression.
-*Regex* | Regex to filter or capture specific parts of the names return by your data source query. Optional.
-*Sort* | Define sort order for options in dropdown. **Disabled** means that the order of options returned by your data source query will be used.
-
-#### Using regex to filter/modify values in the Variable dropdown
+### Query
 
-Using the Regex Query Option, you filter the list of options returned by the Variable query or modify the options returned.
+Query variables allow you to write a data source query that might return a list of metric names, tag values, keys, server names, sensor IDs, or data centers.
 
-Examples of filtering on the following list of options:
+For instructions, refer to [Add a query variable]({{< relref "add-query-variable.md" >}}).
 
-```text
-backend_01
-backend_02
-backend_03
-backend_04
-```
-
-##### Filter so that only the options that end with `01` or `02` are returned:
-
-Regex:
-
-```regex
-/.*[01|02]/
-```
+### Custom
 
-Result:
+Define the variable options manually using a comma-separated list.
 
-```text
-backend_01
-backend_02
-```
-
-##### Filter and modify the options using a regex capture group to return part of the text:
+### Text box
 
-Regex:
+Text box variables display as a free text input field with an optional default value.
 
-```regex
-/.*(01|02)/
-```
+### Constant
 
-Result:
-
-```text
-01
-02
-```
+Define a hidden constant. Useful for metric path prefixes for dashboards you want to share. During dashboard export, constant variables are made into an import option.
 
-#### Filter and modify - Prometheus Example
-
-List of options:
-
-```text
-up{instance="demo.robustperception.io:9090",job="prometheus"} 1 1521630638000
-up{instance="demo.robustperception.io:9093",job="alertmanager"} 1 1521630638000
-up{instance="demo.robustperception.io:9100",job="node"} 1 1521630638000
-```
-
-Regex:
-
-```regex
-/.*instance="([^"]*).*/
-```
-
-Result:
-
-```text
-demo.robustperception.io:9090
-demo.robustperception.io:9093
-demo.robustperception.io:9100
-```
+### Data source
 
-### Query expressions
+Data source variables allow you to quickly change the data source for an entire dashboard. Useful if you have multiple instances of a data source in for example different environments.
 
-The query expressions are different for each data source.
+### Interval
```

Comment:
Why draft of interval page, seems like this has the same content as the new page?

Reply:
Because once I get the initial template figured out, there will be a separate task topic for each variable type.